src/components/home/Landing.jsx

```diff
@@ -1,5 +1,53 @@
+import Image from "next/image";
+import logo from "@/public/prishi-logo.webp";
+import landing from "@/public/landing.webp";
+import Button from "@/components/Button";
+import Link from "next/link";
+
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen ">
+      <div className="flex flex-col w-7/12 justify-center gap-3 pl-12">
+        <div className="flex justify-center">
+          <Image src={logo} alt="rishi-logo" className="w-24 h-auto" />
+        </div>
+
+        <div className="flex justify-center">
+          <div className="font-urbanist font-extrabold text-rishi-black text-5xl">
+            Project RISHI at UCR
+          </div>
+        </div>
+
+        <div className="flex justify-center">
+          <div className="font-normal text-2xl text-rishi-green text-center">
+            Dedicated to promoting sustainable
+            <p className="break-before-column">
+              growth in the small village Ghangar, India
+            </p>
+          </div>
+        </div>
+
+        <div className="flex justify-center py-3">
+          <Link
+            href="https://www.instagram.com/ucrprishi/?hl=en"
+            target="_blank"
+          >
+            <Button buttonText={"Join Us"} />
+          </Link>
+        </div>
+      </div>
+
+      <div className="flex relative w-screen -top-24 -right-24">
```

Comment:
justify-center items-center so the image shrinks towards the middle when you shrink the screen
(adjust -top-24 -right-24 to look the same as it does now)